docs/flk

```diff
@@ -476,6 +563,12 @@ TOKENIZE () {
     local idx=0
     local chunk=0
     local chunksz=500
+
+    if ((datalen>500))
```

Comment:
It's a bug, when the word is too big the parse break... I got it when the read-string need to get something bigger than 500 chars